pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -51,7 +51,7 @@ type BrokerSpec struct {
 
 	// AuthInfo contains the data that the service catalog should use to authenticate
 	// with the Broker.
-	AuthInfo *BrokerAuthInfo `json:"authInfo,omitempty"`
```

Comment:
If `authInfo` is required, we will need a validation to reject a broker without an authInfo.